src/sentry/static/sentry/app/views/groupDetails/pluginExtras.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const PluginExtras = React.createClass({
```

Comment:
Yeah, I think it's generally a best practice to use `const` to signify that a variable can't ever or shouldn't ever be changed. The top answer [here](http://programmers.stackexchange.com/questions/278652/how-much-should-i-be-using-let-vs-const-in-es6) is a pretty good (slightly) longer explanation.